session/middleware/start_session.go

```diff
@@ -50,8 +50,12 @@ func StartSession() http.Middleware {
 		req.Next()
 
 		// Save session
-		if err := s.Save(); err != nil {
+		if err = s.Save(); err != nil {
 			color.Red().Printf("Error saving session: %s\n", err)
 		}
+
+		// Release session
+		// TODO - any better way to release the session?
+		session.SessionFacade.(*session.Manager).ReleaseSession(s.(*session.Session))
```

Comment:
Add the `ReleaseSession` method to interface.

```suggestion
		session.SessionFacade.ReleaseSession(s)
```